proto/cluster.proto

```diff
@@ -166,34 +167,36 @@ message Cluster {
   Environment environment = 5;
   
   // Required field. This field indicates ray cluster configuration
-  ClusterSpec cluster_spec = 6;
+  ClusterSpec cluster_spec = 6 [(google.api.field_behavior) = REQUIRED];
+ 
   // Optional. Annotations, for example, "kubernetes.io/ingress.class" to define Ingress class 
-  map<string, string> annotations = 7;
+  map<string, string> annotations = 7 [(google.api.field_behavior) = OPTIONAL];
```

Comment:
Do we really need to specify optional? Everything in PB is optional by default.
There is optional in many places

Reply:
It's a matter of taste, it can be removed as it doesn't seem to influence the generated swagger doc that might be used to generate a client.
@tedhtchang  what is your take ?

Reply:
I agree we marking at least REQUIRED and OUTPUT.